web/jobs/jobs.go

```diff
@@ -320,15 +316,8 @@ func getTriggerState(c echo.Context) error {
 	if err != nil {
 		return wrapJobsError(err)
 	}
-
 	if err = middlewares.Allow(c, permission.GET, t); err != nil {
-		// For konnector trigger, we have a specific logic to check for
-		// permissions: if the application permissions and the konnector
-		// permissions intersect on one or more doctypes, we give the right to the
-		// application to read the trigger state.
```

Comment:
Can you add a paragraph about this change in https://github.com/cozy/cozy-stack/blob/master/docs/important-changes.md please?

Reply:
Done with 64b8bf1a0ed2fcefc09e3369d5840bf713db3fd9